server_admin/topics/authentication/webauthn.adoc

```diff
@@ -244,3 +244,52 @@ The final configuration of the flow looks similar to this:
+
+{project_name} permits administrators to configure the `WebAuthn Passwordless Policy` in a way that allows loginless authentication. Please note that loginless authentication can only be configured with `WebAuthn Passwordless Policy` and with `WebAuthn Passwordless` credentials. WebAuthn loginless authentication and WebAuthn passwordless authentication can be configured on the same realm but will share the same policy `WebAuthn Passwordless Policy`.
+
+===== Setup
+
+Set up WebAuthn Loginless support as follows:
+
+. Register a new required action for WebAuthn passwordless support. Use the steps described in <<_webauthn-register, Enable WebAuthn Authenticator Registration>>. Register the `Webauthn Register Passwordless` action.
+
+. Configure the `WebAuthn Passwordless Policy` . Perform the configuration in the Admin Console, `Authentication` section,  in the tab `WebAuthn Passwordless Policy`. You have to set the *User Verification Requirement* to *Required* and the *Require Resident Key* to *Yes* when you configure the policy for loginless scenario. Please note that since there isn't a dedicated Loginless policy it won't be possible to mix authentication scenarios with user verification=no/resident key=no and loginless scenarios (user verification=yes/resident key=yes). Please also note that storage capacity is usually very limited on security keys meaning that you won't be able to store many resident keys on your security key.
+
+. Configure the authentication flow. Create a new authentication flow, add the "WebAuthn Passwordless" execution and set the Requirement setting of the execution to `Required`
+
+The final configuration of the flow looks similar to this:
+
+image:images/webauthn-loginless-flow.png[]
+
+You can now add the required action `WebAuthn Register Passwordless` to a user, already known to {project_name}, to test this. The user with the required action configured will have to authenticate (with a username/password for example) and will then be prompted to register a security key to be used for loginless authentication.
+
+===== Vendor specific remarks
+
+====== Compatibility check list
+
+Loginless authentication with {project_name} requires the security key to meet the following features
+
+** FIDO2 compliance: not to be confused with FIDO/U2F
+** User verification: the ability for the security key to authenticate the user (prevents someone finding your security key to be able to authenticate loginless and passwordless)
+** Resident key: the ability for the security key to store the login and the cryptographic keys associated with the client application
+
+====== Windows Hello
+
+In order to use Windows Hello based credentials to authenticate against {project_name} you have to configure the `WebAuthn Passwordless Policy` to include the `RS256` signature algorithm.
```

Comment:
Always omit "In order" which adds no meaning.
add a comma at the end of the introductory clause - after {project_name}
Avoid "you have to" or "you must," which sound dictatorial and add no meaning. Simply start with action: configure.